CHANGELOG.md

```diff
@@ -104,6 +104,9 @@
 * [#3299](https://github.com/rubocop-hq/rubocop/issues/3299): `Lint/UselessAccessModifier` now warns when `private_class_method` is used without arguments. ([@Darhazer][])
 * [#6026](https://github.com/rubocop-hq/rubocop/pull/6026): Exclude `refine` by default from `Metrics/BlockLength` cop. ([@kddeisz][])
 * [#4882](https://github.com/rubocop-hq/rubocop/issues/4882): Use `IndentationWidth` of `Layout/Tab` for other cops. ([@AlexWayfer][])
+* [#5843](https://github.com/bbatsov/rubocop/issues/5843): Add configuration options to `Naming/MemoizedInstanceVariableName` cop to allow leading underscores. ([@leklund][])
+* [#5843](https://github.com/bbatsov/rubocop/issues/5843): Add `EnforcedStyleForLeadingUnderscores` to `Naming/MemoizedInstanceVariableName` cop to allow leading underscores. ([@leklund][])
+* Add new `Style/MultilineMethodSignature` cop. ([@drenmi][])
```

Comment:
It seems that CI has failed because git merge driver did merge as unexpected for changelog.
https://travis-ci.org/rubocop-hq/rubocop/jobs/423855586#L1509-L1538

Reply:
Thanks @koic! Will fix.